lte/gateway/c/oai/include/amf_config.h

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/*****************************************************************************
+  Source      amf_message.h
+  Date        2020/07/28
+  Author      Ashish Prajapati
+  Subsystem   Access and Mobility Management Function
+  Description Defines Access and Mobility Management Messages
+*****************************************************************************/
+#pragma once
+
+#include <pthread.h>
+#include <stdint.h>
+#include <arpa/inet.h>
+#include <stdlib.h>
+#include "amf_default_values.h"
+#include "common_types.h"
+#include "3gpp_23.003.h"
+#include "3gpp_24.008.h"
+#include "log.h"
+#include "service303.h"
+
+#define MIN_GUAMFI 1
+#define MAX_GUAMFI 5
+
+typedef uint64_t imsi64_t;
+typedef uint32_t amf_ue_ngap_id_t;
+
+typedef struct m5g_served_tai_s {
```

Comment:
please comment on the use of the data structure and the fields

Reply:
done